SPECS/dracut/dracut.spec

```diff
@@ -56,6 +56,18 @@ Patch:          0014-fix-systemd-pcrphase-in-hostonly-mode-do-not-try-to-include
 Patch:          0015-fix-systemd-pcrphase-make-tpm2-tss-an-optional-dependency.patch
 Patch:          0016-Handle-SELinux-configuration-for-overlayfs-folders.patch
 Patch:          avoid-mktemp-collisions-with-find-not-path.patch
+# fix-dracut-systemd-include-systemd-cryptsetup.patch has been introduced  
+# by the Azure Linux team to ensure that the systemd-cryptsetup module is included  
+# in the initramfs when needed.  
+# In dracut version 102, the systemd-cryptsetup module was split from the crypt module  
+# and is no longer included by default in initramfs. This causes encrypted volumes  
+# defined in crypttab to not be processed during the initrd phase.  
+# This patch modifies the dracut-systemd module to explicitly include systemd-cryptsetup  
+# when required, restoring expected cryptsetup functionality.  
+# This is a temporary fix until Dracut is upgraded to 104+.  
```

Comment:
This somewhat promises we would be upgrading Dracut. I would ease this and suggest that the patch can be removed if the version was v104.